string priority
geometry_msgs/Pose2D position
string robot_name